Army/ArmyLRFull.cs

```diff
@@ -137,159 +136,94 @@ public void LR()
-                if (item.Coins)
-                    Core.AddDrop(Reward);
-                else
-                    Core.Logger($"{item} has been excluded as it is not a AC item.");
-            }
-        }
-        else Core.AddDrop(Loot);
+            Core.EquipClass(ClassType.Farm);
+            Adv.BestGear(GearBoost.dmgAll);
+            if (!Core.CheckInventory("Tethered Soul", 300))
+                Core.SellItem("Tethered Soul", 0, true);
+            Core.KillMonster("revenant-999999", "r2", "Left", "*", "Tethered Soul", 300, false); //Temp fix for players > 3
+            //ArmyHunt("revenant",  new[] {"Forgotten Soul"}, "Tethered Soul", false, 300);
 
-        Core.EquipClass(ClassType.Farm);
-        Core.RegisterQuests(questID);
+            if (!Core.CheckInventory("Darkened Essence", 500))
+                Core.SellItem("Darkened Essence", 0, true);
+            ArmyHunt("shadowrealmpast", new[] { "Pure Shadowscythe, Shadow Guardian, Shadow Warrior" }, "Darkened Essence", ClassType.Farm, false, 500);
 
-        Army.SmartAggroMonStart(map, Monsters);
+            Bot.Quests.UpdateQuest(2061);
+            Adv.BestGear(GearBoost.Undead);
+            if (!Core.CheckInventory("Dracolich Contract", 1000))
+                Core.SellItem("Dracolich Contract", 0, true);
+            ArmyHunt("necrodungeon", new[] { "5 Headed Dracolich" }, "Dracolich Contract", ClassType.Farm, false, 1000);
 
-        while (!Bot.ShouldExit)
-            Bot.Combat.Attack("*");
-        Army.AggroMonStop(true);
+            Bot.Drops.Pickup("Revenant's Spellscroll");
+            Core.Logger($"Completed x{i++}");
+        }
+        Core.CancelRegisteredQuests();
     }
 
-    public void GetItem(string map = null, string Monster = null, int questID = 000, string item = null, bool isTemp = false, int quant = 1)
+    public void ArmyLF3(int quant)
     {
-        Core.PrivateRooms = true;
-
-        Quest QuestData = Core.EnsureLoad(questID);
-        ItemBase[] RequiredItems = QuestData.Requirements.ToArray();
-        ItemBase[] QuestReward = QuestData.Rewards.ToArray();
-
-        Core.AddDrop(item);
-        Core.EquipClass(ClassType.Farm);
-        if (!Bot.Quests.Active.Contains(QuestData))
-            Core.EnsureAccept(questID);
-
-        Army.SmartAggroMonStart(map, Monster);
-
-        while (!Bot.ShouldExit && !Core.CheckInventory(item, quant))
-            Core.HuntMonster(map, Monster);
-        Army.AggroMonStop(true);
+        Core.Logger($"Farming {quant} Exalted Crown");
+        while (!Bot.ShouldExit && !Core.CheckInventory("Exalted Crown", 10))
+        {
+            Core.RegisterQuests(6899);
+            Core.BuyItem("underworld", 216, "Hooded Legion Cowl");
+            /*This is the only not prefarmed item left to get*/
+            ArmyDarkTokenOfDage(100);
+            if (!Core.CheckInventory("Legion Token", 4000))
+                ArmyLTs(4000);
+            Bot.Drops.Pickup("Exalted Crown");
```

Comment:
just do a Core.AddDrop(if it isnt in one already,) and a Bot.Wait.ForPickup("item"); at the end